<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DXF-PostGIS Конвертер</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>601</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>DXF → SQL</string>
    </attribute>
    <widget class="QPushButton" name="open_dxf_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Открыть DXF</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>240</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>9</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Файл не выбран :(</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QTreeWidget" name="dxf_tree_widget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>671</width>
       <height>421</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>671</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>671</width>
       <height>421</height>
      </size>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>2</string>
      </property>
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="select_area_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выбрать область</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="coord">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>401</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="type_shape">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>прямоугольник</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>круг</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>полигон</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="type_selection">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>внутри</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>снаружи</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>пересечение</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Тип фигуры</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Правило выбора</string>
     </property>
    </widget>
    <widget class="QPushButton" name="export_to_db_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Экспорт в БД</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="export_to_file_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Экспорт в файл</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="filter_groupbox">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>271</width>
       <height>110</height>
      </rect>
     </property>
     <property name="title">
      <string>Фильтр выделения</string>
     </property>
     <widget class="QLabel" name="layer_filter_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Оставить выделение только для слоёв:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="layer_filter_list">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>44</y>
        <width>200</width>
        <height>61</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="apply_filter_button">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>40</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>ОК</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_filter_button">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>40</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Сброс</string>
      </property>
     </widget>
    </widget>
    <zorder>coord</zorder>
    <zorder>open_dxf_button</zorder>
    <zorder>label</zorder>
    <zorder>dxf_tree_widget</zorder>
    <zorder>select_area_button</zorder>
    <zorder>type_shape</zorder>
    <zorder>type_selection</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>export_to_db_button</zorder>
    <zorder>export_to_file_button</zorder>
    <zorder>filter_groupbox</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>SQL → DXF</string>
    </attribute>
    <widget class="QLabel" name="settings_structureLabel_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Базы данных</string>
     </property>
    </widget>
    <widget class="QTreeWidget" name="db_structure_treewidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>681</width>
       <height>521</height>
      </rect>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Настройки</string>
    </attribute>
    <widget class="QCheckBox" name="enable_logging_checkbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Включить логи</string>
     </property>
    </widget>
    <widget class="QComboBox" name="language_combo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ru</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>en</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="language_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Язык интерфейса</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
